proto/tendermint/state/types.proto

```diff
@@ -77,4 +77,7 @@ message State {
 
   // the latest AppHash we've received from calling abci.Commit()
   bytes app_hash = 13;
+
+  // timeouts to be used for the next block height
+  tendermint.abci.TimeoutsInfo timeouts = 15 [(gogoproto.nullable) = false];
```

Comment:
[question] why not 14? The last used field number is `13`.

Reply:
this is a good question

Reply:
It's weirdly formatted but above there is `initial_height` which uses the 14 tag